### PR TITLE
Restore custom constraints to ec2 module

### PR DESCRIPTION
**Pull request by dfellis**
While reading through the code to see how to add custom constraints for the load balancer entity, I realized that the custom constraints in the EC2 module had been accidentally dropped in a migration a couple of versions ago, so this restores that back.

### CODE REVIEW

**Review by depombo — APPROVED** [review]

**Review by aguillenv — APPROVED** [review]

File: src/modules/0.0.14/aws_ec2/sql/create_custom_constraints.sql [hunk 1]
```diff
@@ -30,11 +30,11 @@ $$;
 ALTER TABLE instance ADD CONSTRAINT check_role_ec2 CHECK (role_name is NULL OR (role_name is NOT NULL AND check_role_ec2(role_name)));
 
 -- Create EC2 instance and EBS volume must be in the same availability zone
-create or replace function check_instance_ebs_availability_zone(_instance_id integer, _ebs_availability_zone general_purpose_volume_availability_zone_enum) returns boolean
+create or replace function check_instance_ebs_availability_zone(_instance_id integer, _ebs_availability_zone character varying) returns boolean
```
Comment (aguillenv) [inline src/modules/0.0.14/aws_ec2/sql/create_custom_constraints.sql:33]: hmmm why this change?

Comment (dfellis) [reply src/modules/0.0.14/aws_ec2/sql/create_custom_constraints.sql:33]: The availability zone is no longer an enum, it's a table that gets joined in now.

Comment (aguillenv) [reply src/modules/0.0.14/aws_ec2/sql/create_custom_constraints.sql:33]: ohh!

**Review by aguillenv — APPROVED** [review]
